controllers/os-common/charts/templates/rbac.yaml

```
@@ -0,0 +1,67 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gardener-extension-os-{{ .Values.OperatingSystem.Name }}
+  labels:
+    app.kubernetes.io/name: gardener-extension-os-{{ .Values.OperatingSystem.Name }}
+    helm.sh/chart: gardener-extension-os-{{ .Values.OperatingSystem.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+rules:
+- apiGroups:
+  - extensions.gardener.cloud
+  resources:
+  - operatingsystemconfigs
+  - operatingsystemconfigs/status
```

I'll move this in a separate rule
```suggestion
- apiGroups:
  - extensions.gardener.cloud
  resources:
  - operatingsystemconfigs/status
  verbs:
  - patch
  - update
```

as `Status` subresources only support `patch` and `update`.

---

As you said, having these extra verbs are not needed, but neither do they harm and adding one additional rule add some verbosity.